junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/NullEnum.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2015-2019 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.jupiter.params.provider;
+
+/**
+ * Dummy enum class to be used as default value of annotation attributes.
+ *
+ * @since 5.6
+ * @see EnumSource#value()
+ */
+enum NullEnum {
```

Comment:
I see you decided to make this package private.

Hope that doesn't cause any issues.

In the very least, it would make the Javadoc for `NullEnum` unavailable if package-private types are not included in the generated Javadoc.

Reply:
> Hope that doesn't cause any issues.

What issues would that cause?

> In the very least, it would make the Javadoc for NullEnum unavailable if package-private types are not included in the generated Javadoc.

You'd make it unavailable if package-private types _are_ included, right?

Reply:
> > Hope that doesn't cause any issues.
> 
> What issues would that cause?

I just think it's somewhat unusual to have a non-public type as a default value for an annotation attribute. Other than potentially not publishing the Javadoc for it, I'm not immediately aware of any concrete runtime issues, but perhaps there's some use case we're not thinking of in which a user may wish to refer to the type in compiled user code.

> > In the very least, it would make the Javadoc for NullEnum unavailable if package-private types are not included in the generated Javadoc.
> 
> You'd make it unavailable if package-private types _are_ included, right?

If we leave it "as is" (package private) and do not publish package private members in the generated Javadoc, then the Javadoc would not be published for `NullEnum`.

Reply:
For example, our build configures `memberLevel = JavadocMemberLevel.PROTECTED` which should correlate to:

> Shows only protected and public classes and members.

Reply:
I don't feel strongly about not making it `public`. Thus, I'll do it for the sake of consistency.

Reply:
Done in 5f243c2d0d1c823f4bac006d73cf90332dbcd038.